en/deploy-tiflash.md

```diff
@@ -75,6 +75,11 @@ TiFlash supports mounting multiple Persistent Volumes (PVs). If you want to conf
 >
 > Since TiDB Operator will mount PVs automatically in the **order** of the items in the `storageClaims` list, if you need to add more disks to TiFlash, make sure to append the new item only to the **end** of the original items, and **DO NOT** modify the order of the original items.
 
+TiDB Operator manages TiFlash by creating [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/). Since `StatefulSet` does not support modifying `volumeClaimTemplates` after creation, updating `storageClaims` to add the disk can not mount the additional PV to the Pod. There are two solutions:
```

Comment:
```suggestion
TiDB Operator manages TiFlash by creating [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/). Since `StatefulSet` does not support modifying `volumeClaimTemplates` after creation, updating `storageClaims` to add the disk cannot mount the additional PV to the Pod. There are two solutions:
```